source/content/guides/object-cache/05-troubleshoot-object-cache.md

```diff
@@ -25,7 +25,7 @@ Refer to the [Minimum Cache Lifetime](/drupal-cache#minimum-cache-lifetime) sect
 
 ### Cannot Activate the Redis Plugin for WordPress
 
-WP Redis is a drop-in plugin that should only be loaded using the installation methods in [Enable Object Cache](/guides/object-cache/enable-object-cache). No activation is required.
+WP Redis is a drop-in plugin that should only be loaded using the installation methods in [Enable Object Cache](/guides/object-cache/enable-object-cache), and only if you are not using Object Cache Pro. No activation is required.
```

Comment:
I'm not sure this sentence is entirely true but I don't know enough about wp redis to argue— but I find it hard to believe the plugin doesn't need to be active at all? Maybe that's true?

Reply:
Hrmmm.... my phrasing here was ambiguous. The longer explanation: WP Redis and OCP do both have drop-in plugins, but they are called different things. The WP Redis drop in should not be present if you're using OCP, and vice versa. I'll try to clean that up.

Reply:
Let me know if the updated phrasing makes more sense.

Reply:
Well, they both have a drop-in plugin in wp-content, what I was curious about is the statement that "No activation is required."